NOnion/Utility/Base64Util.fs

```diff
@@ -0,0 +1,21 @@
+﻿namespace NOnion.Utility
+
+module Base64Util =
+    let private Modulus = 4
+    let private PaddingCharacter = "="
```

Comment:
private fields should be camelCase, not PascalCase

Reply:
Fixed